src/AzureFunctions/Properties.xml

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0"?>
+<Content xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Object id="AZURE_FUNCTIONS">
+    <Group>
+      <Id>Cloud</Id>
+      <Name>Cloud Settings</Name>
+      <Description>Settings for Azure Functions deployment</Description>
+      <Type>Main</Type>
+      <Childs>
+	      <Prop>
+          <Id>AZURE_FUNCTIONS_FUNCTION_APP</Id>
+          <Name>Function app</Name>
+          <Description>Name of the Function app in Azure. It must start with a letter and only contain letters, numbers, _, and -, up to 127 characters in length.</Description>
+          <Type>Text</Type>
+          <Default></Default>
```

Comment:
I think this Property is missing option type required.

Option required seems unimplemented in IDE, however, it's better to have the attribute here.

Reply:
I'll fix it.
With respecto to the merge, Ana said:
Ese proyecto no tiene TRUNK/STABLE. Cuando el PR esté aprobado, cuando hagas el merge automáticamente se hará la integración con TRUNK (SVN).

So, in order to be realeased, I think that it should be done in the NexusDependecies.

Reply:
OK!
So, Trunk will get the change automatically.
Stable will need to change NeXusDependencies.